paper/scripts/run_exp_by_examples.py

```diff
@@ -0,0 +1,294 @@
+import sys
+import os
+from typing import Tuple
+sys.path.append('..')
+
+import torch # type: ignore
```

Comment:
why type ignore?

Reply:
Removed. I don't remember, but think it was for Visual Studio IDE (to remove some unuseful warnings)